### PR TITLE
solo2-cli 0.2.1 (new formula)

### PR DESCRIPTION
**Pull request by b-reich**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by b-reich** [comment]
new version of #107574

**Comment by b-reich** [comment]
Any update here?

File: Formula/solo2-cli.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class Solo2Cli < Formula
+  desc "Solo 2 library and CLI in Rust"
+  homepage "https://docs.rs/solo2"
+  url "https://github.com/solokeys/solo2-cli/archive/refs/tags/v0.2.1.tar.gz"
+  sha256 "ae9ef9dd174a8b8294941635a3a66dc9062fd4b595d5f1f6507b5a5a232d6932"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
```
Comment (SMillerDev) [inline Formula/solo2-cli.rb:8]: Where is this used?

File: Formula/solo2-cli.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  def install
+    system "cargo", "install", "--all-features", *std_cargo_args
+
+    bash_completion.install "target/release/solo2.bash"
+    fish_completion.install "target/release/solo2.fish"
+    zsh_completion.install "target/release/_solo2"
+  end
+
+  test do
+    assert_match 'cmd+="__version"', shell_output("#{bin}/solo2 completion bash")
```
[the remark this answers is not in the document]
Comment (b-reich) [reply Formula/solo2-cli.rb:27]: just a test for an example output of the tool.

Without any hardware dongle you dont get much output there.

File: Formula/solo2-cli.rb (hunk 1 above)
Comment (b-reich) [reply Formula/solo2-cli.rb:8]: you are right
May some copy paste mistake

**Pull request closed without merging**